src/nvim/mbyte.c

```diff
@@ -1837,10 +1837,11 @@ int mb_off_next(const char_u *base, const char_u *p)
   return i;
 }
 
-/// Return the offset from "p" to the last byte of the character it points
-/// into.  Can start anywhere in a stream of bytes.
-/// Composing characters are not included.
-int mb_tail_off(const char *base, const char *p_in)
+/// Return the offset from `p_in` to the last byte of the codepoint it points
+/// to.  Can start anywhere in a stream of bytes.
+/// Composing characters are counted separately, not included in the character.
+/// Note: Counts individual codepoints of composed characters separately.
```

Comment:
One of these is enough; take your pick ;)

(My personal preference of course is the one that matches `utf_cp_head_off` below.)

Reply:
Missed that. Thanks.